src/assets/scss/settings/_settings.variables.scss

```diff
@@ -0,0 +1,7 @@
+$cursors: default, not-allowed, wait;
+$inspector-column-max-width: 265px;
+$controls-column-max-width: 265px;
+$controls-column-compressed-max-width: 95px;
+$toolbar-height: 2rem;
+$vertex-error-color: #ED4757;
+$controls-transition: 0.3s;
```

Comment:
In principle I think this is good but I think there are a few issues in practice. A lot of these variables are only used in a single component and I think there's value in those cases in having them declared close to where they're used. I think the controls and inspector variables should be moved into the components that refer to them, and this should be easier with the refactoring that I've just merged.

I think you could go further with the `$controls-transition` though. The transition duration is being used globally and is currently duplicated. You could rename this variable `$transition-duration` and refactor to use it everywhere.